sda-commons-dependencies/build.gradle

```diff
@@ -15,8 +15,8 @@ ext {
   weldVersion = '3.1.5.Final'
   openTracingUtilVersion = '0.33.0'
   jsonUnitVersion = '2.24.0'
-  scalaVersion = '2.12.8' // Always prefer the version from kafka_2.12. Make sure that the major version is always aligned with kafka, s3mock, and mbknor-jackson-jsonschema.
-  kafkaVersion = '2.2.1' // Fix version as decided in https://sda-se.atlassian.net/l/c/ce8EFM2e. Be careful when upgrading this dependency.
+  scalaVersion = '2.12.11' // Always prefer the version from kafka_2.12. Make sure that the major version is always aligned with kafka, s3mock, and mbknor-jackson-jsonschema.
+  kafkaVersion = '2.6.1'
```

Comment:
Why not use `2.7.0`? https://search.maven.org/artifact/org.apache.kafka/kafka_2.12/2.7.0/jar

Reply:
> https://github.com/salesforce/kafka-junit/releases/tag/v3.2.2

Our testing library says: "Validate against Kafka versions 2.5.x and 2.6.x"